package.json

```diff
@@ -33,6 +33,7 @@
     "@hapi/hapi": "18.4.1",
     "@hapi/inert": "5.2.0",
     "@hapi/vision": "5.5.2",
+    "axios": "^0.25.0",
```

Comment:
Needed this package to send requests from the server. Node native `http` module is very cumbersome to use. Ideally I would replace frontend `ajaxService` with `axios` too (or `window.fetch`), probably the first time we need some advanced features, like cancelling a server request.